microsoft-edge/webview2/reference/javascript/hostobjectsoptions.yml

```diff
@@ -96,6 +96,20 @@ properties:
       content: 'log: (...data: any[]) => void;'
       return:
         type: '(...data: any[]) =&gt; void'
+  - name: shouldPassTypedArraysAsArrays
```

Comment:
i'm not finding this "should" item rendered at the review url that's at topmost comment of PR
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/javascript/hostobjectsoptions?branch=main&branchFallbackFrom=pr-en-us-3181

Reply:
The build probably hadn't finished for `pr-en-us-3181` then. I see it now.

Reply:
i'm still not seeing it, after powerful clearing cache/refresh.  will continue trying & post here...